packages/api/lib/request.js

```diff
@@ -91,20 +91,18 @@ function validateGranuleExecutionRequest(req, res, next) {
       }
       return true;
     });
-  }
-
-  if (payload.query
+  } else {
```

Comment:
This logic was broken, as in all current instances using this validator, if you have a granules object specified the Metrics query isn't needed and shouldn't be validated, however the query object (e.g. `limit`)still changes the configuration for the RDS query.